custom_components/ics_calendar/filter.py

```diff
@@ -121,4 +121,4 @@ def filter_event(self, event: CalendarEvent) -> bool:
         :return: true if the event should be included, otherwise false
         :rtype: bool
         """
-        return self.filter(event.summary, event.description)
+        return self.filter(event.summary, event.description or '')
```

Comment:
I'd rather see filter() modified to handle None, instead of passing an empty string to it. The parsers call filter() directly, not just filter_event(). Just passing an empty string here doesn't guarantee a fix.

Reply:
That makes perfect sense. I'll handle `None` in `_is_match()`.